allennlp/service/server_sanic.py

```diff
@@ -6,8 +6,8 @@
 
 # Move access.log and error.log to /tmp by default
 # If someone really wants them, they can move them back
-LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
-LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
+# LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
+# LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
```

Comment:
why did you comment these out? they were there because sanic was creating these files in the project root every time you ran the unit tests